content/actions/hosting-your-own-runners/about-self-hosted-runners.md

```diff
@@ -49,7 +49,7 @@ For more information about installing and using self-hosted runners, see "[Addin
 
 You can use any machine as a self-hosted runner as long at it meets these requirements:
 
-* You can install and run the self-hosted runner application on the machine. For more information, see "[Supported operating systems for self-hosted runners](#supported-operating-systems-for-self-hosted-runners)."
+* You can install and run the self-hosted runner application on the machine. For more information, see "[Supported operating systems for self-hosted runners](#supported-architectures-and-operating-systems-for-self-hosted-runners)."
```

Comment:
```suggestion
* You can install and run the self-hosted runner application on the machine. For more information, see "[Supported architectures and operating systems for self-hosted runners](#supported-architectures-and-operating-systems-for-self-hosted-runners)."
```
👋 @dschaller Thanks so much for opening this PR and fixing this link! While we're here, let's also update the name of the section we're sending folks to. I've suggested that above ☝️ Once you've accepted it, let me know and I'll get this merged in ⚡